test/IRGen/virtual-function-elimination-ir.swift

```diff
@@ -48,31 +48,43 @@ func func2() {
-  // CHECK:  call { ptr, i1 } @llvm.type.checked.load(ptr [[SLOT]], i32 0, metadata !"$s4main7MyClassC3fooyyFTq")
+  // CHECK:  call { ptr, i1 } @llvm.type.checked.load(ptr [[SLOT]], i32 0, metadata !"$s4main14MyDerivedClassC3fooyyFTq")
 }
 
-// CHECK-64: !0 = !{i64 56, !"$s4main7MyClassC3fooyyFTq"}
-// CHECK-64: !1 = !{i64 64, !"$s4main7MyClassC3baryyFTq"}
-// CHECK-64: !2 = !{i64 72, !"$s4main7MyClassCACycfCTq"}
-// CHECK-64: !3 = !{i64 1, i64 56, i64 76}
-// CHECK-64: !4 = !{}
-// CHECK-64: !5 = !{i64 80, !"$s4main7MyClassC3fooyyFTq"}
-// CHECK-64: !6 = !{i64 88, !"$s4main7MyClassC3baryyFTq"}
-// CHECK-64: !7 = !{i64 96, !"$s4main7MyClassCACycfCTq"}
-// CHECK-64: !8 = !{i64 1, i64 80, i64 100}
-// CHECK-64: !9 = !{i64 68, !"$s4main7MyClassC3baryyFTq"}
-// CHECK-64: !10 = !{i64 80, !"$s4main7MyClassCACycfCTq"}
-// CHECK-64: !11 = !{i64 1, i64 56, i64 84}
+// CHECK-64: !0 =  !{i64 [[MyDerivedClassCMnFooOffset:56]], !"$s4main7MyClassC3fooyyFTq"}
+// CHECK-64: !1 =  !{i64 64, !"$s4main7MyClassC3baryyFTq"}
+// CHECK-64: !2 =  !{i64 72, !"$s4main7MyClassCACycfCTq"}
+// CHECK-64: !3 =  !{i64 1, i64 56, i64 76}
+// CHECK-64: !4 =  !{}
+// CHECK-64: !5 =  !{i64 [[MyDerivedClassCMfFooOffset:80]], !"$s4main7MyClassC3fooyyFTq"}
+// CHECK-64: !6 =  !{i64 [[MyDerivedClassCMfBarOffset:88]], !"$s4main7MyClassC3baryyFTq"}
+// CHECK-64: !7 =  !{i64 [[MyDerivedClassCMfInitOffset:96]], !"$s4main7MyClassCACycfCTq"}
+// CHECK-64: !8 =  !{i64 1, i64 80, i64 100}
+// CHECK-64: !9 =  !{i64 [[MyDerivedClassCMnFooOffset]], !"$s4main14MyDerivedClassC3fooyyFTq"}
+// CHECK-64: !10 = !{i64 [[MyDerivedClassCMnBarOffset:68]], !"$s4main14MyDerivedClassC3baryyFTq"}
+// CHECK-64: !11 = !{i64 [[MyDerivedClassCMnBarOffset]], !"$s4main7MyClassC3baryyFTq"}
+// CHECK-64: !12 = !{i64 [[MyDerivedClassCMnInitOffset:80]], !"$s4main14MyDerivedClassCACycfCTq"}
+// CHECK-64: !13 = !{i64 [[MyDerivedClassCMnInitOffset]], !"$s4main7MyClassCACycfCTq"}
+// CHECK-64: !14 = !{i64 1, i64 56, i64 84}
+// CHECK-64: !15 = !{i64 [[MyDerivedClassCMfFooOffset]], !"$s4main14MyDerivedClassC3fooyyFTq"}
+// CHECK-64: !16 = !{i64 [[MyDerivedClassCMfBarOffset]], !"$s4main14MyDerivedClassC3baryyFTq"}
+// CHECK-64: !17 = !{i64 [[MyDerivedClassCMfInitOffset]], !"$s4main14MyDerivedClassCACycfCTq"}
 
-// CHECK-32: !0 = !{i64 56, !"$s4main7MyClassC3fooyyFTq"}
-// CHECK-32: !1 = !{i64 64, !"$s4main7MyClassC3baryyFTq"}
-// CHECK-32: !2 = !{i64 72, !"$s4main7MyClassCACycfCTq"}
-// CHECK-32: !3 = !{i64 1, i64 56, i64 76}
-// CHECK-32: !4 = !{}
-// CHECK-32: !5 = !{i64 52, !"$s4main7MyClassC3fooyyFTq"}
-// CHECK-32: !6 = !{i64 56, !"$s4main7MyClassC3baryyFTq"}
-// CHECK-32: !7 = !{i64 60, !"$s4main7MyClassCACycfCTq"}
-// CHECK-32: !8 = !{i64 1, i64 52, i64 64}
-// CHECK-32: !9 = !{i64 68, !"$s4main7MyClassC3baryyFTq"}
-// CHECK-32: !10 = !{i64 80, !"$s4main7MyClassCACycfCTq"}
-// CHECK-32: !11 = !{i64 1, i64 56, i64 84}
+// CHECK-32: !0 =  !{i64 [[MyDerivedClassCMnFooOffset:56]], !"$s4main7MyClassC3fooyyFTq"}
```

Comment:
I've only tested 64 bit locally, so I could have made an error here. I hope swift-ci runs the 32 bit version of the test.